AUTHORS

```diff
@@ -0,0 +1,7 @@
+Dong Ahn
+Al Chu
+Ryan Day
+Jim Garlick
+Mark Grondona
+Stephen Herbein
+Christopher Moussa
```

Comment:
We got rid of the authors file in flux-core because it's a pain to maintain, and duplicates data in the commit history.  The relevant  PR is flux-framework/flux-core#3090.  Just need an autmake option to make it OK with the authotools.

Reply:
Oh! I didn't know about this - thank you for letting me know. I'll go ahead and change that first commit to just add the `foreign` keyword to automake.